src/Testcontainers/Containers/TestcontainersContainer.cs

```diff
@@ -87,6 +87,11 @@ public string Hostname
     {
       get
       {
+        if (!string.IsNullOrEmpty(TestcontainersSettings.DockerHostOverride))
```

Comment:
@eddumelendez @kiview @mdelapenya How do you favour the host override configuration? Is that similar to your language implementation?